src/main/java/ruslan/shastkiv/bookstore/model/Category.java

```diff
@@ -0,0 +1,31 @@
+package ruslan.shastkiv.bookstore.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+
+@Entity
+@Table(name = "categories")
+@Getter
+@Setter
+@SQLDelete(sql = "UPDATE categories SET is_deleted = TRUE WHERE id =?")
+@SQLRestriction("is_deleted=false")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false)
+    private String name;
+    private String description;
+    @Column(nullable = false)
+    @JsonIgnore
+    private boolean isDeleted = false;
```

Comment:
Why do we need @JsonIgnore for this field? We don't want to return a `Category ` model from our controllers as a response or part of it